collector/tcpstat.go

```diff
@@ -0,0 +1,210 @@
+// +build !notcpstat
+
+package collector
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	procTcpStat  = "/proc/net/tcp"
```

Comment:
`ss` uses Netlink sockets under the hood. From `strace`:

Request for data:

```
socket(PF_NETLINK, SOCK_RAW, 4)         = 3
sendmsg(3, {msg_name(12)={sa_family=AF_NETLINK, pid=0, groups=00000000}, msg_iov(1)=[{"L\0\0\0\22\0\1\3@\342\1\0\0\0\0\0\2\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0"..., 76}], msg_controllen=0, msg_flags=0}, 0) = 76
```

Reading data:

```
recvmsg(3, {msg_name(12)={sa_family=AF_NETLINK, pid=0, groups=00000000}, msg_iov(1)=[{"`\0\0\0\22\0\2\0@\342\1\0\2127\0\0\n\10\0\0\2408\2w\0\0\0\0\0\0\0\0"..., 8192}], msg_controllen=0, msg_flags=0}, 0) = 1344
```

Go seems to support Netlink (http://golang.org/pkg/syscall), so this should be doable... let me know if I can help build that code.